metadata/elasticsearch/src/main/java/com/spotify/heroic/metadata/elasticsearch/MetadataBackendKV.java

```diff
@@ -236,12 +236,26 @@ public Groups groups() {
                 try (Scope ignored = tracer.withSpan(span)) {
                     span.putAttribute("index", AttributeValue.stringAttributeValue(index));
 
-                    if (!writeCache.acquire(Pair.of(index, series.getHashCodeTagOnly()),
-                        reporter::reportWriteDroppedByCacheHit)) {
-                        span.setStatus(
-                            Status.ALREADY_EXISTS.withDescription("Write dropped by cache hit"));
-                        span.end();
-                        continue;
+                    if (indexResourceIdentifiers) {
```

Comment:
Is it maybe possible to use a function or a variable like this?
`HashCode hc = indexResourceIdentifiers ? series.getHashCode() : series.getHashCodeTagOnly()`